commit 25ee292b205e6085ba624b8ac8a45fe11a99e5cc
Author: Cristian One»õ <onet.cristian@gmail.com>
Date:   Tue Apr 10 20:25:26 2012 +0300

    The kmymoney_base target must depend on the kmm_config config target to avoid build failures when aggresive parallel make is used.
    
    BUG: 297008

diff --git a/kmymoney/widgets/CMakeLists.txt b/kmymoney/widgets/CMakeLists.txt
index 20d910d..5c6d28d 100644
--- a/kmymoney/widgets/CMakeLists.txt
+++ b/kmymoney/widgets/CMakeLists.txt
@@ -89,6 +89,8 @@ ADD_CUSTOM_TARGET(generate_base_ui_srcs DEPENDS
 # We can compile the uncritical sources without KMM_DESIGNER flags
 KDE4_ADD_LIBRARY(kmymoney_base STATIC ${_uncritial_common_sources})
 
+ADD_DEPENDENCIES(kmymoney_base kmm_config)
+
 ########### QtDesigner Widget Library (kmymoneywidgets) #################
 # we never link against this library,
 # but it is needed for uic and QtDesigner
